apps/masterbots.ai/components/routes/chat/chat.tsx

```
@@ -45,20 +45,21 @@ import { useModel } from "@/lib/hooks/use-model";
   AiClientType,
-  ChatLoadingState,
   ChatProps,
-  CleanPromptResult,
+  CleanPromptResult
 } from "@/types/types";
+import { WordWareFlowPaths } from "@/types/wordware-flows.types";
 import type { ChatRequestOptions, CreateMessage } from "ai";
 import { type Message, useChat } from "ai/react";
 import { useScroll } from "framer-motion";
 import { uniqBy } from "lodash";
+import { Thread } from "mb-genql";
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)